src/ontology/mondo-edit.obo

```diff
@@ -480916,13 +480909,17 @@ name: choroideremia hypopituitarism
 def: "This is an X-linked recessive retinal degenerative disease that leads to degeneration of the choriocapillaris, the retinal pigment epithelium, and the photoreceptor of the eye. Hypopituitarism is the decreased (hypo) secretion of one or more of the eight hormones normally produced by the pituitary gland at the base of the brain." []
 subset: gard_rare {source="MONDO:GARD"}
 subset: rare
+synonym: "CHM-hypopituitarism syndrome" EXACT [Orphanet:1434]
 synonym: "choroideraemia co-occurrent with hypopituitarism" RELATED []
-synonym: "choroideraemia hypopituitarism" RELATED []
-synonym: "choroideremia co-occurrent with hypopituitarism" RELATED []
-synonym: "choroideremia hypopituitarism" EXACT []
+synonym: "choroideremia-hypopituitarism syndrome" EXACT [MONDO:0015429]
```

Comment:
this is an obsolete name from Orphanet, so I don't think we should add this as a synonym. I usually don't xrefs that point back to obsolete terms either, I add the xref from the original source (but since this term is obsolete, I don't think we should include it as a synonym).

Or we could include it with an xref to Orphanet and note it is a historic synonym that is no longer used (there is an annotation property for that- DEPRECATE, see: https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#synonym-types)

Reply:
removed this synonym